computer-use-demo/computer_use_demo/loop.py

```diff
@@ -62,6 +62,15 @@ class APIProvider(StrEnum):
 </IMPORTANT>"""
 
 
+def debug_print(*args):
+    """Print debug information only if DEBUG_MODE environment variable is set to 'true'."""
+    if os.getenv("DEBUG_MODE", "false").lower() == "true":
+        print(*args)
+
+
+MAX_RETRIES = 42  # Maximum number of retries for rate limit errors
```

Comment:
this is a _very large_ number of retries. In addition, the SDK already has an [automatically used and configurable option](https://github.com/anthropics/anthropic-sdk-python?tab=readme-ov-file#retries) for retries.